microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -222,4 +222,11 @@ FIPS mode::
 ==== Enhancements
 
 Adds OLM image references in dedicated file::
-* Previously, image references for embedding MicroShift OLM in a RHEL for Edge image were part of the `microshift-olm` RPM package. The application package had to be downloaded and run to retrieve the information. With this release, image references are now in the dedicated `microshift-olm-release-info` RPM package for easier use. (link:https://issues.redhat.com/browse/OCPBUGS-29246[OCPBUGS-29246])
\ No newline at end of file
+* Previously, image references for embedding MicroShift OLM in a RHEL for Edge image were part of the `microshift-olm` RPM package. The application package had to be downloaded and run to retrieve the information. With this release, image references are now in the dedicated `microshift-olm-release-info` RPM package for easier use. (link:https://issues.redhat.com/browse/OCPBUGS-29246[OCPBUGS-29246])
+
+[id="microshift-4-15-3-dp"]
+=== RHBA-2024:1257 - {microshift-short} 4.15.3 bug fix and enhancement update advisory
+
+Issued: 2024-03-20
+
```

Comment:
@ShaunaDiaz This LGTM. 

Obviously I'm not familiar with your product or the precise release process your team uses, but I noticed one advisory is accessible and the other is not. I actually expected neither to be available, so I just wanted to mention what I noticed in case both should be.

Reply:
Thanks, @rh-tokeefe, yeah, we won't merge until both are released. There was a problem yesterday with some of the releases that ART was working on.

Reply:
Oh, and MicroShift shares both release cadence and numbering with OpenShift, btw.